Authentication-System/authen/app/views.py

```diff
@@ -0,0 +1,56 @@
+from django.shortcuts import render, redirect
+from django.contrib.auth import login,logout,authenticate
+from django.contrib import messages
+from .models import CustomUser
+from django.http import JsonResponse
+
+def signup(request):
+    if request.method == "POST":
+        username = request.POST.get("username")
```

Comment:
optimise this code instead of picking every variable in a single line.

Reply:
I am picking only those variable in a single line. where i need to add restriction like get phone numbers to add a restriction must be  unique

Reply:
`email = request.POST.get("email")
        contact_number = request.POST.get("contact_number")
        years_of_experience = request.POST.get("years_of_experience")
        university_name = request.POST.get("university_name")
        degree_name = request.POST.get("degree_name")
        password1 = request.POST.get("password1")`
        
        Since variable name and the name received in request are same, we can optimise the code to pick all variables using one line only.

Reply:
yes updated